docs/changelog-fragments.d/README.md

```diff
@@ -0,0 +1,79 @@
+<!-- markdownlint-disable first-line-heading -->
+
+(_myst_parser_adding_changelog_fragments)=
+
+## Adding change notes with your PRs
```

Comment:
Here's a list of recommendations pasted from other projects I've contributed a similar setup to (namely, like setuptools/aiohttp/ansible-navigator/ansible-pylibssh etc). It could be improved over time.

Note that this document is also rendered on GitHub when people navigate to the folder via web UI.